sites/all/modules/_custom/NC_models/static/nonClassUpdates.php

```diff
@@ -1,6 +1,6 @@
 <?php
-//define('DRUPAL_ROOT', '/Applications/XAMPP/htdocs/dragoon-lms');
-define('DRUPAL_ROOT', '/home/laits/public_html/');
+define('DRUPAL_ROOT', '/Applications/XAMPP/htdocs/dragoon-lms');
```

Comment:
@riteshsamala  Could we use universal paths instead of hard-coding this?  Why is needed anyway?